patches/ungoogled-chromium/compiler_specific.patch

```diff
@@ -0,0 +1,11 @@
+--- a/base/compiler_specific.h
++++ b/base/compiler_specific.h
+@@ -334,7 +334,7 @@ inline constexpr bool AnalyzerAssumeTrue(bool arg) {
+ 
+ // Use nomerge attribute to disable optimization of merging multiple same calls.
+ #if defined(__clang__) && __has_attribute(nomerge) && !defined(OS_CHROMEOS)
+-#define NOMERGE [[clang::nomerge]]
```

Comment:
Dont know why, but dont compile without this patch.

Reply:
Apparently related to the compilation with llvm11, not needed with llvm12.

Should I updat to 12 in this PR, or keep this patch and make a second PR for the udpate to version 12?
What do you prefere?
Note that since the version 12 is not in the standard debian or ubuntu repository (need to double check, but I assume so), I would have to install it with the apt repositories of llvm.org directly.

Reply:
I think version 11 is fine.